src/validators/concretes/validationCondition.ts

```diff
@@ -0,0 +1,23 @@
+import { ValidationException } from "../../exceptions";
+
+export class ValidationCondition<T> {
+    public static isNotNullNorUndefined<K>(object: K): boolean {
+        return object !== null && object !== undefined ? true : false;
```

Comment:
replace with `return (object !== null && object !== undefined);`

Reply:
This whole validation method is wrong right now as an `object` can never be `null` and `undefined` at the same time. As the method name ( isNotNull**Nor**Undefined ) suggests the expression should be:
```ts
return (object !== null || object !== undefined) ? true : false;
```